src/h5web/visualizations/MappedHeatmapVis.tsx

```diff
@@ -16,7 +18,19 @@ function MappedHeatmapVis(props: Props): ReactElement {
   assertArray<number>(value);
 
   const dataArray = useMappedArray(dataset, value, mapperState);
-  return <HeatmapVis dataArray={dataArray} />;
+
+  const { resetDomains, ...config } = useHeatmapConfig(
+    ({ dataDomain, ...rest }) => rest, // eslint-disable-line @typescript-eslint/no-unused-vars
+    shallow
+  );
```

Comment:
I am afraid I don't understand this piece of code...

Reply:
Sorry, I was about to write a comment. I'm excluding `dataDomain` from the state selection to save a render. (Technically, it saves a render only if the user hadn't selected a custom domain.)